template.html

```diff
@@ -11,6 +11,7 @@
 				<th style="text-align: center;">Result</th>
 				<th style="text-align: center;">Bugzilla</th>
 				<th style="text-align: center;">Jira</th>
+				<th style="text-align: center;">Other</th>
```

Comment:
Does it make sense to consolidate into one column? That'll help the width of this column stay sane and can  specially help when the report shows up in an email. 

You can prepend the markers with "BZ", "JIRA", or "Ref" (to indicate "other") - WDYT?

Reply:
I do like the idea of reevaluating the report format, but I think I want to wait until we get the Ceph container info in before doing so.